biztalk/technical-guides/test-results-biztalk-server-key-performance-indicators.md

```diff
@@ -18,16 +18,16 @@ manager: "anneta"
 This topic summarizes BizTalk Server Key Performance Indicators (KPI) observed during the test scenarios. Specifically these tests evaluated throughput as measured by the "**BizTalk:Messaging/Documents processed/Sec**" performance monitor counter, and latency, as measured by the Visual studio client response time.
 
 ## Summary of BizTalk Server Key Performance Indicators
- For each scenario the physical machines were restricted so that number of logical processors and virtual processors was equivalent. This was done using the /maxmem and /numproc boot.ini switches. For more information about using these switches, see “Boot INI Options Reference” at [https://go.microsoft.com/fwlink/?LinkId=122139](/previous-versions/bb963892(v=msdn.10)).
+ For each scenario the physical machines were restricted so that number of logical processors and virtual processors was equivalent. This was done using the /maxmem and /numproc boot.ini switches. For more information about using these switches, see [Boot INI Options Reference](https://go.microsoft.com/fwlink/?LinkId=122139.).
```

Comment:
```suggestion
For each scenario the physical machines were restricted so that number of logical processors and virtual processors was equivalent. This was done using the **/maxmem** and **/numproc** switches for the **boot.ini** file. For more information, see [Boot Options in a Boot.ini File](/windows-hardware/drivers/devtest/boot-options-in-a-boot-ini-file).
```